<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MikuniWidget</class>
 <widget class="QWidget" name="MikuniWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="menuGroup">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Menu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MenuWidget" name="menuPage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="troubleCodeGroup">
     <property name="title">
      <string>Trouble Code</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="TroubleCodeWidget" name="troubleCodePage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="liveDataGroup">
     <property name="title">
      <string>Live Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="LiveDataWidget" name="liveDataPage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="versionGroup">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>ECU Version</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="softwareLabel">
          <property name="text">
           <string>Software</string>
          </property>
          <property name="buddy">
           <cstring>softwareVersion</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="softwareVersion"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="hardwareLabel">
          <property name="text">
           <string>Hardware</string>
          </property>
          <property name="buddy">
           <cstring>hardwareVersion</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hardwareVersion"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="waveFromsGroup">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="PlotWidget" name="plotPage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuWidget</class>
   <extends>QWidget</extends>
   <header>menuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TroubleCodeWidget</class>
   <extends>QWidget</extends>
   <header>troublecodewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LiveDataWidget</class>
   <extends>QWidget</extends>
   <header>livedatawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
